bin/genesis

```diff
@@ -799,14 +799,23 @@ create_normal_environment() {
 EOF
 	done
 
-	local director_uuid
-	director_uuid=$(bosh status --uuid)
-	if [[ "$?" -eq 0 ]] ; then
-		cat <<EOF > ${root}/director.yml
+	# As a convenience, try to get the BOSH Director UUID, but only if the director in set and reachable.
+	local director_uuid='(( param "Please specify the BOSH Director UUID in '${root}'/director.yml" ))'
+	local director_url
+	local director_address
+	local fetched_director_id
+
+	read director_url director_address < <(bosh target 2>/dev/null | sed -e 's/.* is \(https\{0,1\}:\/\/\(.*\):[0-9]\{1,\}\) (.*)$/\1 \2/')
```

Comment:
Can we just read ~/.bosh_config via jq?  Do we want to do that?  Is it more or less stable than CLI output?